.github/workflows/setup-anvil.yml

```diff
@@ -19,8 +19,24 @@ jobs:
           version: "latest"
       - name: Tunnel Anvil
         id: tunnel
-        run: cloudflared tunnel --url http://localhost:8545 &
+        run: cloudflared tunnel --url http://localhost:8545 --logfile cloudflared.log &
         shell: bash
+      - name: Extract Cloudflared URL
+        run: |
+          grep -E "https://.*.trycloudflare.com" cloudflared.log | cut -d'|' -f2 | tr -d ' ' > cloudflare-url.txt
+          cat cloudflare-url.txt
+        shell: bash
+      - name: Store URL
```

Comment:
Do we still need this step? Along with "Create comment"

Reply:
@rndquu it is up to you and @pavlovcik to decide. I added this to prove my QA, create an artifact and add comment to JIRA, can safely remove both now.

Reply:
Pls remove

Reply:
done